man/man7/zfsprops.7

```diff
@@ -1285,7 +1291,9 @@ or zvol blocks into the special allocation class.
 Blocks smaller than or equal to this
 value will be assigned to the special allocation class while greater blocks
 will be assigned to the regular class.
-Valid values are zero or a power of two from 512 up to 1048576 (1 MiB).
+Valid values are from 0 to maximum block size (
+.Ar 16 MiB
+).
```

Comment:
Let's add a phrase here to make it clear this threshold is applied after any compression.